### DISCUSSION TITLE
礼问Umbra音量插件问题

### DISCUSSION THREAD

**Discussion by Lao9u** [DeepseaXX/aakaigua#7] [Q&A]
如题，礼问老师有没有遇到Umbra里明明提高了音量插件里的整体音量，结果下次打开游戏的时候音量还是调整前的（

**Comment by DeepseaXX** [comment]
没有遇到过耶

**Reply by Lao9u** [reply]
感谢佬的回复，我自己排查出问题了是有不兼容问题。另外如果可以的话想请问佬关于gatherbuddyreborn的问题。就我个人想根据采集笔记里还没挖过的内容批量导入方便采集，但好像gbr本身只能一个材料一个材料的选，请问佬能够指教一下怎么创建一个预设吗？

**Reply by DeepseaXX** [reply]
> 感谢佬的回复，我自己排查出问题了是有不兼容问题。另外如果可以的话想请问佬关于gatherbuddyreborn的问题。就我个人想根据采集笔记里还没挖过的内容批量导入方便采集，但好像gbr本身只能一个材料一个材料的选，请问佬能够指教一下怎么创建一个预设吗？

没试过……但是你可以通过结合teamcraft，导出/导入列表试试看

**Comment by DeepseaXX** [comment]
坏了，哈哈，我也遇到了
怀疑是由于关游戏的时候没正常保存导致的，比如使用了某些直接杀死游戏的插件
